abb_irb120_moveit_config/package.xml

```diff
@@ -0,0 +1,40 @@
+<package>
+  <name>abb_irb120_moveit_config</name>
+  <version>0.2.0</version>
+  <description>
+    <p>
+      MoveIt package for the ABB IRB 120.
+    </p>
+    <p>
+      An automatically generated package with all the configuration and launch
+      files for using the ABB IRB 120 with the MoveIt Motion Planning
+      Framework.
+    </p>
+  </description>
+  <author email="assistant@moveit.ros.org">MoveIt Setup Assistant</author>
+  <maintainer email="culletom@tcd.ie">Mark Culleton (Trinity College Dublin)</maintainer>
+
+  <license>BSD</license>
+
+  <url type="website">http://moveit.ros.org/</url>
+  <url type="bugtracker">https://github.com/ros-industrial/abb/issues</url>
```

Comment:
I'm not sure, but perhaps this should point to the issue tracker of the `abb_experimental` repository? All other experimental packages do so, I believe.